redirects.yaml

```
@@ -1,3 +1,16 @@
+# ledgerstatefix docs
+/docs/ledger-state-fix:
+  to: 'https://xrpl.org/docs/references/protocol/transactions/types/ledger-state-fix'
```

This is probably the wrong URL (transaction types are not kebab case). Also, the quotes here are optional too.

Suggested fix: 

```suggestion
  to: https://xrpl.org/docs/references/protocol/transactions/types/ledgerstatefix
```

We should wait until https://github.com/XRPLF/xrpl-dev-portal/pull/2971 is deployed to confirm though